test/PlaidClientTest.js

```diff
@@ -77,11 +77,28 @@ describe('plaid.Client', () => {
     });
   });
 
-  it('can create item add tokens', () => {
-    pCl.createItemAddToken((err, successResponse) => {
+  it('can create item add tokens', cb => {
```

Comment:
This test looks like it was broken previously - it passed immediately as no callback was supplied, and then any failures would be randomly assigned to whatever test was running.